pkg/install/install.go

```diff
@@ -148,6 +148,10 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 	if err != nil {
 		return err
 	}
+	err = i.checkQuota(ctx)
```

Comment:
check quota synchronously on the front end, not asynchronously on the back end.  The quota code should be part of the dynamic validator.